tests/fail-dep/libc/libc_epoll_block_two_thread.rs

```diff
@@ -1,7 +1,7 @@
 //@compile-flags: -Zmiri-preemption-rate=0
 //~^ERROR: deadlocked
 //~^^ERROR: deadlocked
-//@only-target: linux
+//@only-target: linux illumos
```

Comment:
Shouldn't this include `android` as well?

Reply:
Yeah, it should.